spec/mailers/emailer_spec.rb

```diff
@@ -167,7 +165,7 @@
 
     it { should deliver_to("invitee@agency.com") }
     it { should have_subject(/\[Search.gov\] Welcome to Search.gov/) }
-    it { should have_body_text(/https:\/\/localhost:3000\/complete_registration\/some_special_token\/edit/) }
```

Comment:
see question above - seems like we should still be directing them to the `complete_registration` page